test/constraint/spec.test.ts

```diff
@@ -952,8 +952,8 @@ describe('constraints/spec', () => {
     });
   });
 
-  describe('omitTableWithOcclusion', () => {
-    it('return false for raw plot with both x and y as dimensions.', () => {
+  describe('omitTableWithOcclusionIfAutoAddCount', () => {
+    it('return false for raw plot with both x and y as dimensions without autocount or aggregation.', () => {
```

Comment:
Instead of adding random testing in `query.ts`, why don't you add a proper test here and in `generate.test.ts`?